requirements.in

```diff
@@ -1,7 +1,6 @@
 backoff
 dockerfile-parse>=0.0.13
-flatpak-module-tools >= 0.11,!=0.13,<=0.13.1;python_version<"3.9"
-flatpak-module-tools >= 0.11;python_version>="3.9"
+flatpak-module-tools==0.14
```

Comment:
why `==` ? I'd like to receive security updates

Reply:
updated